spec/helpers/post_helper_spec.rb

```diff
@@ -4,11 +4,11 @@
 
 describe PostHelper do
   describe '#tweet_link' do
-    context 'returns a link to twitter' do
+    context 'when returns a link to twitter' do
       specify 'with appropriate data' do
```

Comment:
specify is coming from older days of rspec, maybe it's worth refactoring it to `it`